Python.gitignore

```diff
@@ -72,6 +72,10 @@ target/
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# IPython
+default_profile/
```

Comment:
I'm not clear on where this rule is being used in the documentation linked to - is this supposed to be `profile_default`?

Reply:
Sorry for the typo. It should be `profile_default`, and I've updated my branch.